dist/vis.js

```diff
@@ -1590,7 +1590,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /***/ (function(module, exports, __webpack_require__) {
 
   var require;/* WEBPACK VAR INJECTION */(function(module) {//! moment.js
-  //! version : 2.29.4
+  //! version : 2.29.3
```

Comment:
Hi, 

I am not sure about this hook. I have not changed this code, this might have got changed when I run the build command.
I am using node version 8.11.1 for the build.
Is it something that I need to change ?

Reply:
try build using node version 8.15.1